components/progressTracker/theme.css

```diff
@@ -64,9 +64,39 @@
     }
   }
 
-  &:not(.is-active):not(.is-completed) {
-    .progress-step__label {
-      opacity: 0.5;
+  .status-bullet {
+    background-color: transparent;
+    transition: background-color .1s var(--animation-curve-default) calc(var(--animation-duration) - .1s);
+    align-self: center;
+    border-radius: 50%;
+    display: block;
+    height: 12px;
+    margin: 6px 0;
+    width: 12px;
+    z-index: 2;
+    position: relative;
+    &:before {
+      background-color: transparent;
+      transition: background-color 6s var(--animation-curve-default) calc(var(--animation-duration) - .1s);
+      border-radius: 50%;
+      content: '';
+      display: block;
+      position: absolute;
+      width: 6px;
+      height: 6px;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+    }
+  }
+
+  .progress-step__label {
```

Comment:
We don't use BEM.

Reply:
Changed the classname

 :ok: by @Kemosabert on 19/06/2018 09:55:06